src/jp/project/index.md

```diff
@@ -1,21 +1,33 @@
 ---
-description: Vue A11y is an open-source community project, where the main objective is to create content focused on Vue.js and bringing maximum accessibility
+title: プロジェクト
+description: Vue A11yは、Vue.jsに最大限のアクセシビリティをもたらす事を目的としたオープンソースプロジェクトです。
+
+meta:
+  - property: 'og:image'
+    content: 'https://vue-a11y.com/img/social-share-jp.png'
+  - name: 'twitter:title'
+    content: プロジェクト
```

Comment:
```suggestion
title: はじめに
description: Vue A11yは、Vue.jsに最大限のアクセシビリティをもたらす事を目的としたオープンソースプロジェクトです。

meta:
  - property: 'og:image'
    content: 'https://vue-a11y.com/img/social-share-jp.png'
  - name: 'twitter:title'
    content: はじめに
```

The title should match the main heading.
https://github.com/vue-a11y/vue-a11y.com/pull/47/files#r563237561